### DISCUSSION TITLE
安装依赖的时候一直报错

### DISCUSSION THREAD

**Discussion by luoxy1001** [umijs/umi#6575] [Q&A]
An unexpected error occurred: "https://registry.npm.taobao.org/@ant-design/icons-svg/download/@ant-design/icons-svg-4.1.0.tgz: ESOCKETTIMEDOUT"
我直接访问这个包地址是可以的

**Comment by xiaohuoni** [comment]
没遇到过。换了几个电脑都是可以的，这种应该是你网络问题。你可以尝试切换个网路尝试一下。或者切回 npm 源，用 yarn 安装，感觉也挺快的。

**Reply by luoxy1001** [reply]
用npm也不行，试过清楚yarn和npm缓存，都不行，npm和yarn源都设置淘宝镜像了，但是还是不行，后来我装了cnpm，用cnpm安装依赖居然可以了

**Reply by xiaohuoni** [reply]
谜

**Reply by xiaohuoni** [reply]
谜
